wiki/Play_Styles/Hover/en.md

```diff
@@ -0,0 +1,18 @@
+---
+tags:
+  - tablet
+  - hovering
+  - play style
+---
+
+<!-- This article is a stub -->
+
+# Hover
+
+*Main Page: [Play Styles](/wiki/Play_Styles)*
+
+Hovering is a term used to describe the act of navigating (or "aiming") a [tablet](/wiki/Glossary#tablet) pen across a tablet by holding the pen above the surface of the tablet while still inside of the active hover zone. Thus you are "hovering" the pen above the tablet. This is a method of pen navigation in opposition to [dragging](/wiki/Play_Styles/Drag). It can be found across players of all skill levels, although it is most often associated with highly skilled players.
```

Comment:
```suggestion
Hovering is a term used to describe the act of navigating (or "aiming") a [tablet](/wiki/Glossary#tablet) pen across a tablet by holding the pen above the surface of the tablet while still inside of the active hover zone. Thus, "hovering" the pen above the tablet. This is a method of pen navigation in opposition to [dragging](/wiki/Play_Styles/Drag). It can be found across players of all skill levels, although it is most often associated with highly skilled players.
```
nitpick, ignore if you want to keep it as-is